src/screens/WebsocketResearchScreen/SignedChatScreen.tsx

```diff
@@ -49,10 +49,7 @@ const SignedChatScreen = () => {
   }, [selectedChannel]);
 
   return (
-    <KeyboardAvoidingView
-      behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
-      style={styles.keyboardAvoidingView}
-      keyboardVerticalOffset={-500}>
+    <View style={styles.keyboardAvoidingView}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `KeyboardAvoidingView` has been replaced with a `View` component. This is likely due to the introduction of the `KeyboardWrapper` component that handles keyboard behavior. Ensure that this change doesn't affect the layout or functionality of the screen.
```diff
-    <KeyboardAvoidingView
-      behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
-      style={styles.keyboardAvoidingView}
-      keyboardVerticalOffset={-500}>
+    <View style={styles.keyboardAvoidingView}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->